cypress/integration/edit.js

```diff
@@ -78,6 +78,29 @@ describe('Edit Page', () => {
         );
     });
 
+    it('should allow to select an item from the AutocompleteInput without showing the choices again after', () => {
+        EditCommentPage.navigate();
+        cy.get(EditPostPage.elements.input('post_id')).clear().type('Sed quo');
```

Comment:
I don't understand: you use the EditPostPage here but you loaded the EditCommentPage just above...

Reply:
It actually shows that those page objects are confusing. In this case, both were instantiated at the top of the test file. They both retrieve the correct item from the page with this `input` function. 

Anyway, *fixed* it